Objects/regions.c

```diff
@@ -959,3 +961,57 @@ PyTypeObject PyRegion_Type = {
     0,                                       /* tp_alloc */
     PyType_GenericNew,                       /* tp_new */
 };
+
+static const char *get_region_name(PyObject* obj) {
+    if (IS_DEFAULT_REGION(obj->ob_region)) {
+        return "Default";
+    } else if (IS_IMMUTABLE_REGION(obj->ob_region)) {
```

Comment:
`IS_DEFAULT_REGION` and `IS_IMMUTABLE_REGION` are macros for cases when the region was already stored in a local variable. If you fetch them from an object it would be better to use `_Py_IsLocal` and `_Py_IsImmutable` respectively:

```suggestion
    if (_Py_IsLocal(obj)) {
        return "Default";
    } else if (Py_IsImmutable(obj))) {
```

Same applies to usages below

Reply:
Thanks! What is the reason for having both `IS_DEFAULT_REGION` and `_Py_IsLocal`? Also, maybe we should rename DEFAULT to LOCAL for consistency?

Reply:
`IS_DEFAULT_REGION()` checks if a `regionmetadata*` is the default region, while `_Py_IsLocal` checks if a `PyObject*` is in the local region. So the difference is the argument type.

The macro should probably be renamed to `IS_LOCAL_REGION`.